include/rdma/fi_eq.h

```diff
@@ -74,8 +72,8 @@ struct fid_wait {
 };
 
 struct fi_mutex_cond {
-	pthread_mutex_t		*mutex;
-	pthread_cond_t		*cond;
+	struct pthread_mutex_t	*mutex;
+	struct pthread_cond_t	*cond;
```

Comment:
actually it allows compilation of user's code without including OS specific runtime (it may be compiled without pthread headers on Linux). osd.h headers are not part of external headers and can't be used in rdma/* files